nucypher/utilities/prometheus/collector.py

```diff
@@ -121,6 +124,13 @@ def _collect_internal(self) -> None:
 
         self.metrics["learning_status"].state('running' if self.ursula._learning_task.running else 'stopped')
         self.metrics["known_nodes_gauge"].set(len(self.ursula.known_nodes))
+        if self.ursula._availability_tracker:
+            if self.ursula._availability_tracker.running:
+                self.metrics["availability_score_gauge"].set(self.ursula._availability_tracker.score)
+            else:
+                self.metrics["availability_score_gauge"].set(-1)
+        else:
+            self.metrics["availability_score_gauge"].set(-1)
```

Comment:
You can simplify this block.

```suggestion
        if self.ursula._availability_tracker and self.ursula._availability_tracker.running:
            self.metrics["availability_score_gauge"].set(self.ursula._availability_tracker.score)
        else:
            self.metrics["availability_score_gauge"].set(-1)
```